src/shogun/evaluation/ClusteringMutualInformation.cpp

```diff
@@ -54,6 +54,9 @@ float64_t CClusteringMutualInformation::evaluate(CLabels* predicted, CLabels* gr
 	float64_t mutual_info = 0;
 	for (index_t i=0; i < n_class; ++i)
 	{
+		/* print progress */
+		SG_PROGRESS(i, 0, n_class+n_class, 1, "EVALUATING MUTUAL INFORMATION: ");
```

Comment:
These strings `EVALUATING MUTUAL INFORMATION: `

Can't we use pretty function for that? Not so sure we should write this manually every time. Should be the `CClassName::method_name` or so

@vigsterkr @lisitsyn